homeassistant/components/light/osramlightify.py

```diff
@@ -197,22 +236,71 @@ def turn_off(self, **kwargs):
             _LOGGER.debug("turn_off requested transition time for light:"
                           " %s is: %s ",
                           self._name, transition)
-            self._light.set_luminance(0, transition)
+            self._luminary.set_luminance(0, transition)
         else:
             transition = 0
             _LOGGER.debug("turn_off requested transition time for light:"
                           " %s is: %s ",
                           self._name, transition)
-            self._light.set_onoff(0)
-            self._state = self._light.on()
-
+        self._luminary.set_onoff(0)
+        self._state = False
         self.schedule_update_ha_state()
```

Comment:
It's not something that we should fix in a platform but instead fix for everyone at once. We already wait 2 seconds for a status update: https://github.com/home-assistant/home-assistant-polymer/blob/master/src/components/entity/ha-entity-toggle.html#L135-L136

Please remove this line.